GDJS/Runtime/pixi-renderers/pixi-filters-tools.js

```diff
@@ -93,11 +96,70 @@ gdjs.PixiFiltersTools._filters = {
         updateParameter: function(filter, parameterName, value) {
             if (parameterName !== 'opacity') return;
 
-            filter.alpha = value;
+            filter.alpha = gdjs.PixiFiltersTools.clampValue(value, 0, 1);
+        },
+    },
+    BlackAndWhite: {
+        makeFilter: function() {
+            var colorMatrix = new PIXI.filters.ColorMatrixFilter();
+            colorMatrix.blackAndWhite();
+            return colorMatrix;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'opacity') return;
+
+            filter.alpha = gdjs.PixiFiltersTools.clampValue(value, 0, 1);
+        },
+    },
+    Noise: {
+        makeFilter: function() {
+            var noise = new PIXI.filters.NoiseFilter();
+            return noise;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'noise') return;
+
+            filter.noise = gdjs.PixiFiltersTools.clampValue(value, 0, 1);
+        },
+    },
+    Blur: {
+        makeFilter: function() {
+            var blur = new PIXI.filters.BlurFilter();
+            return blur;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'blur' &&
+                parameterName !== 'quality' &&
+                parameterName !== 'kernelSize' &&
+                parameterName !== 'resolution') return;
+            
+            if (parameterName === 'kernelSize'){
+                value = gdjs.PixiFiltersTools.clampKernelSize(value);
+            }
+            
+            filter[parameterName] = value;
         },
     },
 };
 
+/**
+ * Enable an effect.
+ * @param {string} filter The name of the filter effect
```

Comment:
This annotation is not right, it should ba a `{PIXI.Filter}` I think and the description should also be updated (`filter The filter to enable or disabled`)